opendatahub-io/odh-dashboard
# Update issues template

Updated issue template with the new `YAML` format to include a more form-oriented bug and feature request panel.

## Testing Instructions
1. Go to [this issue panel](https://github.com/lucferbux/issue_template_mock/issues/new/choose) where I created a mockup version of this PR.
2. Select either *Bug Report* of *Feature Request*.
3. Check the new form.

## Screenshots

<img width="1648" alt="Screenshot 2022-07-07 at 16 31 55" src="https://user-images.githubusercontent.com/16117276/177799329-603dd5d6-3a5e-4c63-8883-2092e9b43766.png">
<img width="1373" alt="Screenshot 2022-07-04 at 10 35 13" src="https://user-images.githubusercontent.com/16117276/177116052-66b00194-b261-4652-971a-3acb70fbcdbf.png">
